pypeit/bspline/setup_package.py

```diff
@@ -55,16 +14,17 @@ def check_for_openmp():
 if not sys.platform.startswith('win'):
     extra_compile_args.append('-fPIC')
 
-if check_for_openmp():
-    extra_compile_args.append('-fopenmp')
-    extra_link_args = ['-fopenmp']
-else:
-    extra_link_args = []
 
 def get_extensions():
-    return [Extension(name='pypeit.bspline._bspline', sources=SRC_FILES,
-                      extra_compile_args=extra_compile_args, language='c',
-                      extra_link_args=extra_link_args,
-                      export_symbols=['bspline_model', 'solution_arrays',
-                                      'cholesky_band', 'cholesky_solve',
-                                      'intrv'])]
+    extension = Extension(name='pypeit.bspline._bspline', sources=SRC_FILES,
+                          extra_compile_args=extra_compile_args, language='c',                      
+                          export_symbols=['bspline_model', 'solution_arrays',
+                                          'cholesky_band', 'cholesky_solve',
+                                          'intrv'])
+    
+    # extension_helpers will check for opnmp support by trying to build
+    # some test code with the appropriate flag. openmp provides a big performance boost, but some
+    # systems, notably apple's version of clang that xcode provides, don't support it out of the box.
+
+    add_openmp_flags_if_available(extension)
+    return [extension]
```

Comment:
looks like missing newline at end of file.

Reply:
Fixed